docs/api-reference/static-analysis.md

```diff
@@ -155,6 +154,44 @@ These plugins are enabled by default:
 
 One custom rule from DBOS, `@dbos-inc/dbos-static-analysis`, is provided in the [`@dbos-inc/eslint-plugin`](https://github.com/dbos-inc/eslint-plugin) package.  This rule is enabled by default.
 
+___
+
+Running a database query with a string that's vulnerable to SQL injection will result in an error.
+This would typically happen inside of a [transaction](https://docs.dbos.dev/tutorials/transaction-tutorial).
+- SQL injection happens when a bad actor puts SQL code as a field into something like an online form,
+and if a programmer builds a raw query from SQL and this data, the bad actor's supposed data may allow them to run
+arbitrary SQL commands over your database.
+- To avoid injection, use prepared statements. But if you accidentally make yourself vulnerable to injection, DBOS is here to save you!
+
+Here's how you should make a prepared statement:
+```ts
+export class Greetings {
+  @Transaction()
+  static async InsertGreeting(ctxt: TransactionContext<Knex>, friend: string, note: string) {
+    await ctxt.client.raw('INSERT INTO greetings (name, note) VALUES (?, ?)', [friend, note]);
+  }
+}
+```
+
+Here's how you would make SQL injection possible:
+```ts
+export class Greetings {
+  @Transaction()
+  static async InsertGreeting(ctxt: TransactionContext<Knex>, friend: string, note: string) {
+    await ctxt.client.raw(`INSERT INTO greetings (name, note) VALUES (${friend}, ${note})`);
+  }
+}
+```
+
+- A LR-value is a literal-reducible value, or a value that is known to be literal at some level.
```

Comment:
Let's remove this paragraph as it's an implementation detail. Replace it with plain language instead.

Reply:
Done